tests/functional/test_service_alias.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import botocore.session
+
+# service_name => alias
+ALIAS_CASES = {
+    'sagemaker-runtime': 'runtime.sagemaker'
```

Comment:
What about making `_SERVICE_NAME_ALIASES` public and dynamically creating the reverse map here?

Otherwise we'll have to remember to update this test if we add a new alias.

Reply:
My initial thought was that having at least a small change to the tests would be good, but I can also see the merit in getting the tests automatically too. And rather than reversion the map I think I'll just make that one public and import it. While it should never happen a service name could have multiple aliases so `alias => service_name` makes more sense.